### ISSUE TITLE
Specify different submit server and skipSegments server

### ISSUE THREAD

**Issue by ajayyy** [ajayyy/SponsorBlock#1559]
To allow using mirrors while still submitting, and prevent leaking private IDs

**Comment by goodness-from-me** [comment]
Some of the read-only mirrors can be found here:
https://github.com/TeamPiped/sponsorblock-mirror

**Comment by d4k0** [comment]
@goodness-from-me 
So if I set one of these read-only mirrors in the settings, I can't submit new segments? Did I understand that correctly?

**Comment by goodness-from-me** [comment]
@d4k0 yes. It will be very hard to maintain database, I think, and to sync it between instances if we're going to allow submitting new segments.

**Comment by d4k0** [comment]
Thanks for the quick answer.
